src/commands/test/lib/run-tests-command.ts

```diff
@@ -137,16 +135,7 @@ export abstract class RunTestsCommand extends AppCommand {
 
         if (!this.async) {
           const exitCode = await this.waitForCompletion(client, testRun.testRunId);
-
-          if (this.testOutputDir) {
-
-              // Download json test result
-              const testReport: TestReport = await client.test.getTestReport(testRun.testRunId, this.app.ownerName, this.app.appName);
-              if (testReport.stats.artifacts) {
-                await downloadUtil.downloadArtifacts(this, this.streamingOutput, this.testOutputDir, testRun.testRunId, testReport.stats.artifacts);
-                await this.mergeTestArtifacts();
-              }
-          }
+          await this.afterCompletion(client, testRun);
```

Comment:
Could we send `this.streamingOutput` here rather than having to make it `protected`? Or have a protected accessor of some kind rather than a variable?

I'm not sure what the normal practice is here with TypeScript (I have my C# head on) so let me know if this is not a problem :)